CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ All notable changes to this project will be documented in this file.
 
 ### :house: (Internal)
 
+* Replace internal references to InstrumentationLibrary with InstrumentationScope [#3202](https://github.com/open-telemetry/opentelemetry-js/pull/3202) @mwear
```

Comment:
I believe this is an internal "housekeeping" change, but wanted to point out that we are adding a property to the `ReadableSpan` interface. Let me know if this should be reclassified.

Reply:
We are changing the public interface here, right? So I'd rather find it can be categorized as a new feature.